### ISSUE TITLE
Rate limiting not working(?)

### ISSUE THREAD

**Issue by kocobee** [Helicone/helicone#2217]
I'm attempting to use helicone as a proxy to add rate limiting, etc. 

I'm able to see logs, and user id's fine, but rate limiting isn't working at all. 

I've hardcoded a user id as "ABC" with a quota of 2 over a period of 1000 seconds, using requests. I'm able to call the OpenAI API each time, with no rate limiting being applied.

My code:

```
const openai = new OpenAI({
    apiKey: process.env.OPENAI_API_KEY,
    baseURL: "https://oai.helicone.ai/v1",
    defaultHeaders: {
      "Helicone-Auth": `Bearer ${process.env.HELICONE_API_KEY}`,
      "Helicone-RateLimit-Policy": `${RL_QUOTA};w=${RL_TIME_WINDOW_SECS};u=${RL_UNIT};s=${RL_SEGMENT}`,
      "Helicone-User-Id": user,
    }
  });
```

**Comment by kocobee** [comment]
cc @colegottdank Any update on this?

**Comment by colegottdank** [comment]
Hi @kocobee, thanks for the report. We are looking into this now! Will get back to you soon.

**Comment by chitalian** [comment]
@kocobee closed with #2230
